docs/legal_assistance.md

```diff
@@ -22,8 +22,8 @@ To apply the suggested configurations in your deployment, update the following f
 - **Azure Semantic Search**: Set `AZURE_SEARCH_USE_SEMANTIC_SEARCH` to `true`
 - **Azure Cognitive Search Top K 15**: Set `AZURE_SEARCH_TOP_K` to `15`.
 - **Azure Search Integrated Vectorization**: Set `AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION` to `true`.
-- **Azure OpenAI Model**: Set `AZURE_OPENAI_MODEL`  to `gpt-4o`.
-- **Azure OpenAI Model Name**: Set `AZURE_OPENAI_MODEL_NAME` to `2024-05-13`.
+- **Azure OpenAI Model Name**: Set `AZURE_OPENAI_MODEL_NAME` to `gpt-4o`. (could be different based on the name of the Azure OpenAI model deployment)
```

Comment:
Should be:  
**Azure OpenAI Model**: Set `AZURE_OPENAI_MODEL` to `gpt-4o`.
**Azure OpenAI Model Name**: Set AZURE_OPENAI_MODEL_NAME to gpt-4o. (could be different based on the name of the Azure OpenAI model deployment)

Reply:
oops, added back.